test/tests/meshgenerators/map_extra_element_ids/tests

```diff
@@ -0,0 +1,11 @@
+[Tests]
+  [map_ids_test]
+    type = RunApp
+    design = 'ElementIDInterface.md'
+    issues = '#25119'
+    input = 'map_extra_element_ids.i'
+    expect_out = 'Mapping from id1 to id2[\s\S]*0 : 1[\s\S]*2 : 1 3'
+    recover = false
+    requirement = "The system shall provide an interface function to get the relation of two extra element integers."
```

Comment:
@loganharbour don't you have another way you like to do a test where the primary outcome of the test is console output?

Reply:
The best option here is to create a test reporter with a JSONDiff. Which we should definitely do here. We've got bitten by managing regex diffs... it's a maintenance nightmare

Reply:
Let me check to see if I can convert this test object to a reporter. I am not very familiar with JSON writing.